### PR TITLE
Add example of pimcore.cnf for MySQL

### PR DESCRIPTION
**Pull request by yariksheptykin**
This MR adds some documentation on how to configure MySQL Server for pimcore using a .cnf file. Options listet in the file can also be helpful while setting up a managed mysql service such as AWS RDS.

### CODE REVIEW

File: doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/05_DB_Setup.md [hunk 1]
```diff
@@ -3,10 +3,38 @@
 Pimcore requires a standard MySQL database, the only thing you should assure is that the database uses `utf8mb4` as character set.  
 If you create a new database just set the character set to `utf8mb4`.
 
+You also might want to set `lower_case_table_names=1` to make sure that tables for pimcore classes are created in lover case even though
```
Comment (NiklasBr) [inline doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/05_DB_Setup.md:6]: "lover case" -> "lower case"

Comment (brusch) [reply doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/05_DB_Setup.md:6]:
```suggestion
You also might want to set `lower_case_table_names=1` to make sure that tables for pimcore classes are created in lower case even though
```

**Comment by brusch** [comment]
@yariksheptykin Thanks a lot for your PR! May I ask for the reason for this PR, because those settings should be applied by Doctrine on connect anyway, so it shouldn't be necessary to explicitly set them again in the mysql config? 
See: https://github.com/pimcore/pimcore/blob/822e68012e49a87859d144c7f7fb8c81a61ea32b/bundles/CoreBundle/Resources/config/pimcore/default.yml#L46

File: doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/05_DB_Setup.md [hunk 2]
```diff
@@ -3,10 +3,38 @@
...
 If you create a new database just set the character set to `utf8mb4`.
 
+You also might want to set `lower_case_table_names=1` to make sure that tables for pimcore classes are created in lower case even though
+their class names contain capital letters.
+
 > Note: You have to create the database manually before you can continue with the web-based installer, 
-> which automatically creates the underlying database schema for Pimcore. 
+> which automatically creates the underlying database schema for Pimcore.
+
+### Dabase server configuration
```
Comment (jdreesen) [inline doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/05_DB_Setup.md:12]:
```suggestion
### Database Server Configuration
```

File: doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/05_DB_Setup.md [hunk 3]
```diff
@@ -3,10 +3,38 @@
...
+
+While setting up the MySQL Server you can enforce `utf8mb4` character set and [lover case table names](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_lower_case_table_names) by placing a pimcore.cnf file with the following contents into the server config directory (e.g. `/etc/mysql/conf.d/`). Refer to server configuration manual applicable to your environment to determine the location of server config directory.
```
Comment (jdreesen) [inline doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/05_DB_Setup.md:14]:
```suggestion
While setting up the MySQL Server you can enforce `utf8mb4` character set and [lower case table names](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_lower_case_table_names) by placing a `pimcore.cnf` file with the following contents into the server config directory (e.g. `/etc/mysql/conf.d/`). Refer to the server configuration manual applicable to your environment to determine the location of the server config directory.
```

File: doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/05_DB_Setup.md [hunk 4]
```diff
@@ -3,10 +3,38 @@
 Pimcore requires a standard MySQL database, the only thing you should assure is that the database uses `utf8mb4` as character set.  
 If you create a new database just set the character set to `utf8mb4`.
 
+You also might want to set `lower_case_table_names=1` to make sure that tables for pimcore classes are created in lower case even though
```
Comment (jdreesen) [inline doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/05_DB_Setup.md:6]:
```suggestion
You also might want to set `lower_case_table_names=1` to make sure that tables for Pimcore classes are created in lower case even though
```

File: doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/05_DB_Setup.md [hunk 5]
```diff
@@ -3,10 +3,38 @@
...
 > Note: You have to create the database manually before you can continue with the web-based installer, 
-> which automatically creates the underlying database schema for Pimcore. 
+> which automatically creates the underlying database schema for Pimcore.
+
+### Dabase server configuration
+
+While setting up the MySQL Server you can enforce `utf8mb4` character set and [lover case table names](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_lower_case_table_names) by placing a pimcore.cnf file with the following contents into the server config directory (e.g. `/etc/mysql/conf.d/`). Refer to server configuration manual applicable to your environment to determine the location of server config directory.
+
+```cnf
+# MySQL Server configuration for pimcore.
```
Comment (jdreesen) [inline doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/05_DB_Setup.md:17]:
```suggestion
# MySQL Server configuration for Pimcore.
```

**Comment by brusch** [comment]
Thanks a lot! 🙏

**Pull request merged**